src/pages/guidelines/opening-link.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Opening link
+description: Opening link in new browser windows and tabs can help users with the browsing experience, especially when they need to compare content or go from one website page to the other back and forth.
+---
+
+<PageDescription>
+
+Opening link in new browser windows and tabs can help users with the browsing experience, especially when they need to compare content or go from one website page to the other back and forth. 
```

Comment:
@JennySanchez you think we can change `go from one website page to the other back and forth.` to `go back and forth from one website page to the other.`? That seems to read more naturally for me

Reply:
Of course, and it makes sense. Thanks, Anna.